flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why? Go runs on aarch64 too

Reply:
The binary is for x86_64, I also don't have an aarch64 machine to test if I did make aarch64 binaries.

Reply:
hence the "build from source".